### PR TITLE
Add button to export submissions data as a CSV file

### PR DESCRIPTION
**Pull request by carlosahs**
The button is added next to the rerun submissions button:

image

Once clicked, a CSV file will be downloaded. The header of the CSV file contains the following attributes:
```ruby
attributes = %w[id uploaded_by_id created_at updated_at test_results_count successful_test_results_count language status]
```

### CODE REVIEW

**Review by carlosahs — COMMENTED** [review]
Includes `ActiveSupport::Testing::TimeHelpers`, otherwise it is reported as an uninitialized constant.

File: app/views/submissions/index.haml [hunk 1]
```diff
@@ -30,7 +40,7 @@
             %td= submission.contributor_names
             %td= submission.status
 
--# reload page every 2s if submissions are still running
-- if @submissions.any?{ |s| !s.checks_completed? }
-  :javascript
-    setTimeout(function() { location.reload() }, 2000)
+-# -# reload page every 2s if submissions are still running
```
Comment (rhomeister) [inline app/views/submissions/index.haml:43]: cleaner to keep this here, and remove from line 16, unless you have a good reason to do so

File: config/routes.rb [hunk 2]
```diff
@@ -29,6 +29,7 @@
   end
 
   resources :assignments do
+    # get 'submissions/export', to: 'submissions#export'
```
Comment (rhomeister) [inline config/routes.rb:32]: remove

**Review by rhomeister — CHANGES_REQUESTED** [review]
2 small comments. Ready to go!

**Pull request merged**